modules/roulette.py

```diff
@@ -0,0 +1,163 @@
+import module
+from discord import Intents, Client, Message, TextChannel
+import globs
+import re 
+import random
+from enum import Enum
+from money import MoneyModule 
+
+MoneyModule = MoneyModule() 
+
+class RouletteModule(module.Module): 
+    def __init__(self):
+        super().__init__()    
+    async def get_res (self, msg: Message): 
+        if "!roulette" not in msg.lower(): 
+            return
+
+        class BetTypes(Enum):
+            STRAIGHT = "straight" 
+            SPLIT = "split"
+            STREET = "street"
+            CORNER = "corner"
+            SIXLINE = "sixline"
+            REDBLACK = "redblack" ## Obligatory 34 joke
+            DOZEN = "dozen"
+            COLUMN = "column"
+            ODDEVEN = "oddeven"
+            HIGHLOW = "highlow"  
+        class BetCalls(Enum): 
+            RED = "red"
+            BLACK = "black" 
+            COLUMN1 = "column1"
+            COLUMN2 =  "column2"
+            COLUMN3 = "column3"
+            DOZEN1 = "first" 
+            DOZEN2 = "second"
+            DOZEN3 = "third" 
+            EVEN = "even"
+            ODD = "odd" 
+            HIGH = "high"
+            LOW = "low"
+        class Error(Enum): 
+            ERROR = "ERROR" 
+        
+        user_id = usr_msg.author.id 
+        if user_id not in MoneyModule.data: MoneyModule.update_balance(user_id, 1000)
+        player_balance = MoneyModule.data[user_id]
+        self.profit = 0
+
+        def parse_bet(message):
+        ## Get information about bet type, player call, and wager while also checking to make sure that the values won't break the roulette wheel.      
+            valid_types =  {"straight", "split", "street", "corner", "sixline", "redblack", "dozen", "column", "oddeven", "highlow"}  
+            valid_calls =  {"red", "black", "column1", "column2", "column3", "first", "second", "third", "even", "odd", "high", "low"} 
+            unpacked_bet = message.lower().split()
+            bet_type = unpacked_bet[1] 
+            if bet_type not in valid_types: return Error.ERROR.value, None, None   

```

Comment:
Instead of returning an invalid state, raise an Exception instead so that the caller can use a try-except block instead of having to do `if bet_type == Error.ERROR.value`.

```python
if bet_type not in valid_types: raise ValueError("Invalid bet type!")
```

Caller:
```python
try:
    (bet_type, player_bet, wager) = parse_bet(player_bet_statement)
except ValueError:
    # Do error handling here
```